cmd/soroban-cli/src/config/locator.rs

```diff
@@ -76,11 +76,11 @@ pub enum Error {
 #[group(skip)]
 pub struct Args {
     /// Use global config
-    #[arg(long)]
+    #[arg(long, global = true)]
```

Comment:
Some commands like `keys add` have a `global` arg. So I curious how this would be handled. In those cases we can thread the global through and use this arg.

Reply:
I don't think it affects in any way. My understanding is that setting `global = true` only means you can use the switch anywhere in the command.

Reply:
Oh duh it's the same `--global` 🤦

Reply:
Moving this PR to draft until this is addressed.

Reply:
Ah yeah sorry. All good. I misunderstood.